Do not redefine gmtime(), localtime()

They are already provided by system time.h header

Upstream-Status: Inactive-Upstream [need a new release]

Sign-off-by: Khem Raj <raj.khem@gmail.com>
--- a/unix/unxcfg.h
+++ b/unix/unxcfg.h
@@ -117,7 +117,6 @@ typedef struct stat z_stat;
 #  endif
 #else
 #  include <time.h>
-   struct tm *gmtime(), *localtime();
 #endif
 
 #if (defined(BSD4_4) || (defined(SYSV) && defined(MODERN)))
